tools/generate_png_images.js

```diff
@@ -6,28 +6,38 @@
   `tools/visual_regression.sh`.
 */
 const { JSDOM } = require('jsdom');
-const dom = new JSDOM(`<!DOCTYPE html></html>`);
-window = dom.window;
-document = dom.window.document;
-
 const fs = require('fs');
-const [scriptDir, imageDir] = process.argv.slice(2, 4);
 
-const Vex = require(`${ scriptDir }/vexflow-debug.js`);
-Vex.Flow.Test = require(`${ scriptDir }/vexflow-tests.js`);
+fs.readFile('build/webComponents-debug.js', 'utf8', function(err, data) { 
+  if (err) {
+    console.log(err)
+  }  
+
+  dom = new JSDOM(`<!DOCTYPE html></html>`, { runScripts: "outside-only" });
+  window = dom.window;
+  document = dom.window.document;
+
+  window.eval(data);
```

Comment:
Its probably worth leaving a comment around here about why we're evaluating the web components source on the window instead of requiring it in Node. Something along the lines of "The web component source contains browser API usage that is not supported in a Node.js environment. It must be run in an environment that more fully implements the DOM."